examples/introduction.ipynb

```diff
@@ -27,19 +27,23 @@
     "    DISPLAY_TYPE,\n",
     ")\n",
     "from string import ascii_uppercase\n",
-    "from random import choice\n",
+    "from random import choice, randint\n",
     "\n",
     "# parameters\n",
     "total_steps = 500\n",
     "timestep = 0.5\n",
     "box_size = 100\n",
-    "n_agents = 5\n",
+    "n_agents = 15\n",
     "min_radius = 5\n",
     "max_radius = 10\n",
     "\n",
-    "type_names = []\n",
-    "for t in range(total_steps):\n",
-    "    type_names.append([choice(ascii_uppercase) for i in range(n_agents)])\n",
+    "types = []\n",
+    "for i in range(n_agents):\n",
+    "    parent = \"test#\" if i % 2 != 0 else \"\"\n",
+    "    types.append(parent + choice(ascii_uppercase))\n",
+    "\n",
+    "type_names = [types for i in range(total_steps)]\n",
+    "\n",
```

Comment:
Just updated the test trajectory show agents with children, and enough total agents that toggling the agents will show overflow behavior of side-panel.